src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("12.0.0.0")]
-[assembly: AssemblyFileVersion("12.0.0.0")]
+[assembly: AssemblyVersion("12.0.1.0")]
+[assembly: AssemblyFileVersion("12.0.1.0")]
```

Comment:
@ritwikbasu can you revert the file version, you don't need to update minor version updates to your file version.
Will create bunch of issue when you have dot releases